superset-frontend/src/dashboard/actions/dashboardState.js

```diff
@@ -321,6 +321,14 @@ export function setDirectPathToChild(path) {
   return { type: SET_DIRECT_PATH, path };
 }
 
+export const SET_IS_TAB_MOUNTED = 'SET_IS_TAB_MOUNTED';
+/**
+ * Set if tab switch animation is in progress
+ */
+export function setIsTabMounted(isAnimating) {
```

Comment:
maybe rename this variable to `isTabMounted` to match the function name?

Reply:
Oops, missed it while search and replace. Thanks for catching this!